src/main/java/seedu/address/ui/ChatWindow.java

```diff
@@ -79,7 +79,7 @@ public void handleSendButtonAction() {
      * @param message The user's input message.
      * @return The assistant's response as a String.
      */
-    private String getResponse(String message) {
+    public String getResponse(String message) {
         message = message.toLowerCase().trim();
 
         switch (message) {
```

Comment:
Good idea for a chat window. Could possibly add cases where the user types in the COMMAND_WORD of each command and the chat returns the MESSAGE_USAGE